modules/rostests/apitests/user32/WndProc.c

```diff
@@ -25,15 +25,12 @@ static LRESULT WINAPI redraw_window_procA(
     HWND hwnd,
     UINT msg,
     WPARAM wparam,
-    LPARAM lparam,
-    DWORD ExtraData,
-    DWORD ExtraData1,
-    DWORD ExtraData2,
-    DWORD ExtraData3)
+    LPARAM lparam)
 {
     switch (msg)
     {
         case WM_PAINT:
+            break;
```

Comment:
Strange hack.

Reply:
The whole test was (is?) a huge hack. At least now it doesn't crash anymore.